megamek/src/megamek/common/MiscType.java

```diff
@@ -11173,19 +11174,36 @@ public static MiscType createLightMinesweeper() {
     public static MiscType createLAMBombBay() {
         MiscType misc = new MiscType();
         misc.name = "Bomb Bay";
-        misc.setInternalName(misc.name);
+        misc.setInternalName(EquipmentTypeLookup.LAM_BOMB_BAY);
         misc.tonnage = 1;
         misc.criticals = 1;
         misc.flags = misc.flags.or(F_BOMB_BAY).or(F_MECH_EQUIPMENT);
         misc.explosive = true;
+        misc.cost = 5000;
         misc.rulesRefs = "110, IO";
-
-        // Not listed in IO; this an amalgam of bimodal and standard LAM values.
+        // IO, p.220/221 + bimodal and standard LAM values
         misc.techAdvancement.setTechBase(TECH_BASE_ALL).setISAdvancement(2680, 2684, DATE_NONE, 3085)
             .setClanAdvancement(DATE_NONE, 2684, DATE_NONE, 2825)
             .setPrototypeFactions(F_TH).setProductionFactions(F_TH)
-            .setTechRating(RATING_D).setAvailability(RATING_D, RATING_E, RATING_F, RATING_F)
-            .setStaticTechLevel(SimpleTechLevel.EXPERIMENTAL);
+            .setTechRating(RATING_D).setAvailability(RATING_B, RATING_E, RATING_E, RATING_E)
```

Comment:
.setISAdvancement(2680, 2684, DATE_NONE, 3085).setClanAdvancement(DATE_NONE, 2684, DATE_NONE, 2825)

Reply:
Don't wish to appear argumentative over a non-issue, but doesn't this contradict IO p.221 directly? (These fuel tanks aren't available for anything but LAMs so they are tied to their availability anyway. I suppose the authors of IO thought a Fuel Tank's just a Fuel Tank)